src/database/migrations/20210504010450-retro-updates.ts

```diff
@@ -0,0 +1,46 @@
+// TODO remove from actual migrations, DataTypes will be used
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { QueryInterface, DataTypes } from 'sequelize';
+
+export default {
+  up: async (queryInterface: QueryInterface): Promise<void> => {
+    // Transactions are automatically use because a namespace is injected into sequelize when fetching configurations
+    await queryInterface.sequelize.transaction(async () => {
+      await queryInterface.addColumn('course_topic_content', 'course_topic_content_grade_ids_that_need_retro', {
```

Comment:
Woof, really trying to hit that 64 character limit. I can see this causing problems for sequelize.

Reply:
Yeah, I was worried about that as well, luckily it will only be a problem if we do a tons of joins and we really need to get rid of the prefix asap